src/pages/workspaces/workspace/Dashboard/Dashboard.test.ts

```diff
@@ -311,7 +347,9 @@ describe('AzureDetails', () => {
     };
 
     // Act
-    const { container } = render(h(AzureStorageDetails, props), { container: document.body.appendChild(document.createElement('dl')) });
+    const { container } = render(h(AzureStorageDetails, props), {
+      container: document.body.appendChild(document.createElement('dl')),
```

Comment:
Do these elements get cleaned up after the test is done?

Reply:
I don't think so, but there appears to be a cleanup method that can be called:

https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
From the linked docs:
> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine)

Reply:
> Unmounts React trees that were mounted with [render](https://testing-library.com/docs/react-testing-library/api/#render).

The `AzureStorageDetails` component and its elements get automatically cleaned up, but I'm not sure about these `dl` elements.

Reply:
It doesn't look like the cleanup function will remove items that were added outside of render.
If I understand correctly, under the covers, the framework mounts a div, renders everything there, then removes the div...

Reply:
This was easy enough to fix by just enclosing the `AzureStorageDetails` in a `dl` normally.